css/webots-doc.css

```diff
@@ -253,3 +253,162 @@
   font-family: monospace;
   width: 80%;
 }
+
+.blog {
+  background-color: #ede9e8;
+  background-image: url(https://www.cyberbotics.com/files/repository/www/wwi/8.5/crossword.png);
```

Comment:
Yes, the 8.5 folder doesn't seem the correct location for a file that is not related to a specific Webots version but is intended to be used in all the blog pages.